candi/bashible/common-steps/node-group/002_abort_if_not_managed_docker.sh.tpl

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if eq .cri "NotManaged" }}
+  {{- if .nodeGroup.cri.notManaged.criSocketPath }}
```

Comment:
additional check for notManaged is exist

Reply:
done